www/_static/swift.css

```diff
@@ -1,64 +1,104 @@
 /* Common styles */
-.body { color: black; background: white; margin: 0 0 0 0 }
+.body {
+  color: black;
+  background: white;
+  margin: 0 0 0 0;
+}
 
 /* No borders on image links */
-a:link img, a:visited img { border-style: none }
+a:link img,
+a:visited img {
+  border-style: none;
+}
 
-address img { float: right; width: 88px; height: 31px; }
-address     { clear: right; }
+address img {
+  float: right;
+  width: 88px;
+  height: 31px;
+}
 
+address {
+  clear: right;
+}
+
+table {
+  border: 2px solid black;
+  border-collapse: collapse;
+  margin-top: 1em;
+  margin-left: 1em;
+  margin-right: 1em;
+  margin-bottom: 1em;
+}
 
-table       { border: 2px solid black;
-  border-collapse: collapse; margin-top: 1em; margin-left: 1em;
-  margin-right: 1em; margin-bottom: 1em; }
-tr, td      { border: 2px solid gray; padding: 4pt 4pt 2pt 2pt; }
-th          { border: 2px solid gray; font-weight: bold; font-size: 105%;
-  background: url("http://llvm.org/img/lines.gif");
-  font-family: "Georgia,Palatino,Times,Roman,SanSerif";
-  text-align: center; vertical-align: middle; }
+tr,
+td {
+  border: 2px solid gray;
+  padding: 4pt 4pt 2pt 2pt;
+}
 
+th {
+  border: 2px solid gray;
+  font-weight: bold;
+  font-size: 105%;
+  background-image: url(http://llvm.org/img/lines.gif);
+  font-family: Georgia,Palatino,Times,Roman,sans-serif;
+  text-align: center;
+  vertical-align: middle;
+}
 
 /* Common for title and header */
-h1, h2 {
-  color: black; background: url("http://llvm.org/img/lines.gif");
-  font-family: "Georgia,Palatino,Times,Roman,SanSerif"; font-weight: bold;
+h1,
+h2 {
+  color: black;
+  background-image: url(http://llvm.org/img/lines.gif);
+  font-family: Georgia,Palatino,Times,Roman,sans-serif;
+  font-weight: bold;
   border-width: 1px;
   border-style: solid none solid none;
   text-align: center;
   vertical-align: middle;
-/*  padding-left: 0pt;*/
   padding-top: 1px;
   padding-bottom: 2px
 }
 
 
 h1 {
-  text-align: center; font-size: 22pt;
-  margin: 20pt 0pt 0pt 0pt; 
+  text-align: center;
+  font-size: 22pt;
+  margin: 20pt 0 0 0;
 }
 
-.doc_title, .title { text-align: left;   font-size: 25pt }
+.doc_title,
+.title {
+  text-align: left;
+  font-size: 25pt;
+}
 
-h2, .doc_subsection { width: 100%;
-  text-align: left;  font-size: 12pt;
+h2,
+.doc_subsection {
+  width: 100%;
+  text-align: left;
+  font-size: 12pt;
   padding: 4pt 4pt 4pt 4pt;
   margin: 1.5em 5em 0.5em 0em
 }
 
 h3 {
-  margin: 2.0em 0.5em 0.5em 0em;
-  font-weight: bold; font-style: oblique;
-  border-bottom: 1px solid #999999; font-size: 12pt;
+  margin: 2em .5em .5em 0;
+  font-weight: bold;
+  font-style: oblique;
+  border-bottom: 1px solid #999999;
+  font-size: 12pt;
   width: 75%;
   clear:both;
 }
 
 body p {
-  margin: 1em 0pt 0pt 15pt; 
+  margin: 1em 0 0 15pt;
 }
 
-
-pre.grammar, pre.stdlib {
+pre.grammar,
+pre.stdlib {
   background: #FFFFE0;
   padding: 1em 0 0 0;
```

Comment:
@coobersdoobers I care. That is a code editing problem, not a css architecture problem :wink: 
I prefer explicitness, but changing shorthand format vs longform format for padding/margin wasn't part of the changes I wanted to make in this pull request. If you'd like to see those changes in I'd suggest opening up a new PR.